tests/actors/rust/wrpc-pinger-component/wit/deps/blobstore/world.wit

```diff
@@ -0,0 +1,5 @@
+package wasi:blobstore@0.2.0-draft;
+
+world imports {
+	import blobstore;
```

Comment:
NIT: We should have a formatter for WIT. Noticed this has tabs whereas WASI landed on a convention of spaces.